modules/aws/awc/main.tf

```diff
@@ -160,6 +169,7 @@ data "aws_iam_policy_document" "awc-policy-doc" {
     resources = [
       "arn:aws:s3:::${var.bucket_name}/${local.provisioning_script}",
       "arn:aws:s3:::${var.bucket_name}/${local.awm_script}",
+      "arn:aws:s3:::${var.bucket_name}/${local.log_manager_script}",
```

Comment:
_Just a cross reference mention here to comment about mixed `local`/`var` usage on line [13](https://github.com/EditShareCo/cloud_deployment_scripts_hp_inc/pull/5/files#diff-51d281d110c60958957a4a5e352dd6e2aa0aa02de13680b3df9564a0dfbe6852R13)_